compose/docker-compose.ape.yml

```diff
@@ -27,6 +27,7 @@ services:
       - external_network
     environment:
       - "PF_MQTT_URL=mqtt://admin:pfadminpw@${PROJECT_PREFIX:-}pf_mosquitto:1883"
+      - "PF_DBSERVER_URL=${PROJECT_PREFIX:-}dbserver"
```

Comment:
usually I try to specify the entire URL string here, not just the hostname (protocol + port too). this makes it work better if I have to specify a URL to a dbserver that lives e.g. outside of docker or on another machine.

example would be `"PF_DBSERVER_URL=http://${PROJECT_PREFIX:-}dbserver:8050"`

cuz if I need to override it to e.g. `"PF_DBSERVER_URL=http://pfubuntu003/api/dbserver"` then it just works!